doc/hyperproof-cli.md

```diff
@@ -0,0 +1,57 @@
+# Hyperproof CLI
```

Comment:
This is all new content.  What we are building right now doesn't completely line up with this but I am working on that separately.